gp-templates/translations.php

```diff
@@ -34,7 +34,7 @@
 		<option value="reject"><?php _e( 'Reject', 'glotpress' ); ?></option>
 		<option value="fuzzy"><?php _e( 'Fuzzy', 'glotpress' ); ?></option>
 	<?php if( $can_write ) : ?>
-		<option value="set-priority"><?php _e( 'Set Priority', 'glotpress' ); ?></option>
+		<option value="set-priority" class="hide-if-no-js"><?php _e( 'Set Priority', 'glotpress' ); ?></option>
```

Comment:
> I think it's safe to say that JS is a requirement for GP

It is. You can't submit any translations without JS because the editor isn't visible. 🙃